packages/rest-api-client/src/KintoneRequestConfigBuilder.ts

```diff
@@ -167,6 +169,24 @@ export class KintoneRequestConfigBuilder implements RequestConfigBuilder {
     }
   }
 
+  private buildProxy(targetProxy?: ProxyConfig): ProxyConfig | undefined {
```

Comment:
IMO: It's better to change the function name and interface as the following:

```suggestion
  private buildProxyConfig(proxyConfig?: ProxyConfig): ProxyConfig | undefined {
```

Because it receives and returns `ProxyConfig`.
I also referred `buildPlatformDependentConfig()` called nearby `buildProxy()`.